tendermint/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name       = "tendermint"
-version    = "0.16.0" # Also update `html_root_url` in lib.rs and
+version    = "0.17.0" # Also update `html_root_url` in lib.rs and
```

Comment:
Don't we need this to be `v0.17.0-rc1` for all crates and references, since it's a [pre-release](https://doc.rust-lang.org/cargo/reference/resolver.html#pre-releases)?

Reply:
I still don't have a good handle on rust releases so I honestly don't know, but I guess so :) 

We should probably be using a tool to bump all these versions together since right now its manual.

Reply:
https://crates.io/crates/cargo-workspaces